env0/data_environment_test.go

```diff
@@ -2,16 +2,29 @@ package env0
 
 import (
 	"regexp"
+	"strconv"
 	"testing"
 
 	"github.com/env0/terraform-provider-env0/client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestEnvironmentDataSource(t *testing.T) {
+	boolean := true
 	environment := client.Environment{
-		Id:   "id-0",
-		Name: "my-environment-1",
+		Id:                          "id-0",
+		Name:                        "my-environment-1",
+		ProjectId:                   "project-id",
+		Status:                      "status",
+		LatestDeploymentLogId:       "latest-deployment-log-id",
+		RequiresApproval:            &boolean,
+		PullRequestPlanDeployments:  &boolean,
+		AutoDeployOnPathChangesOnly: &boolean,
+		ContinuousDeployment:        &boolean,
```

Comment:
since the type is `*bool` which means we are looking for a pointer value,
i had to set the value here to a pointer for a boolean